gollum.gemspec

```diff
@@ -23,18 +23,18 @@ Gem::Specification.new do |s|
   s.rdoc_options = ['--charset=UTF-8']
   s.extra_rdoc_files = %w[README.md LICENSE]
 
-  s.add_dependency 'gollum-lib', '~> 5.0'
+#  s.add_dependency 'gollum-lib', '~> 5.0'
```

Comment:
Why is this line commented out?

Reply:
That's part of requiring my local copy of gollum-lib (for the purpose of testing the octicons). Thanks for catching these!

Reply:
I figured as much! :)